Microsoft.NET.Build.Containers/Layer.cs

```diff
@@ -51,12 +57,14 @@ public static Layer FromFiles(IEnumerable<(string path, string containerPath)> f
         }
 
         string contentHash = Convert.ToHexString(hash).ToLowerInvariant();
+        string uncompressedContentHash = Convert.ToHexString(uncompressedHash).ToLowerInvariant();
 
         Descriptor descriptor = new()
         {
-            MediaType = "application/vnd.docker.image.rootfs.diff.tar", // TODO: configurable? gzip always?
+            MediaType = "application/vnd.docker.image.rootfs.diff.tar.gzip", // TODO: configurable? gzip always?
```

Comment:
Maybe change this to `*.tar.gz` since it's more commonly used? Or is there a reason to use `*.gzip`?

Reply:
These are well-known media types. See https://docs.docker.com/registry/spec/manifest-v2-2/#media-types